library/alloc/src/collections/vec_deque/pair_slices.rs

```diff
@@ -37,7 +36,7 @@ impl<'a, 'b, T> PairSlices<'a, 'b, T> {
     }
 
     pub fn remainder(self) -> impl Iterator<Item = &'b [T]> {
-        array::IntoIter::new([self.b0, self.b1])
+        IntoIterator::into_iter([self.b0, self.b1])
```

Comment:
Shouldn't we generally use this syntax?
```suggestion
        [self.b0, self.b1].into_iter()
```

Reply:
I don't think this would work, the `into_iter()` method resolution for arrays is gated on 2021 edition.